src/libs/ComposerUtils/index.js

```diff
@@ -25,4 +25,18 @@ function canSkipTriggerHotkeys(isSmallScreenWidth, isKeyboardShown) {
     return (isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen()) || isKeyboardShown;
 }
 
-export {getNumberOfLines, updateNumberOfLines, insertText, canSkipTriggerHotkeys};
+/**
+ * Find length of common ending of two strings
+ * @param {String} str1
+ * @param {String} str2
+ * @returns {Number}
+ */
+function getCommonEndingLength(str1, str2) {
```

Comment:
```suggestion
function getFirstDifferencePosition(str1, str2) {
```

Reply:
Resolving in favor of: https://github.com/Expensify/App/pull/22827/files#r1264703411